cluster/libvirt-coreos/user_data_minion.yml

```diff
@@ -17,7 +17,7 @@ coreos:
         --address=0.0.0.0 \
         --hostname-override=${NODE_IPS[$i]} \
         --cluster-domain=cluster.local \
-        --api-servers=http://${MASTER_IP}:8080 \
+        --kubeconfig=/opt/kubernetes/kubeconfig/kubelet.kubeconfig \
```

Comment:
As far as I can tell, `cluster/libvirt-coreos/user_data.yml` contains a unit file that mounts `kubernetes` to `/opt/kubernetes`. In `cluster/libvirt-coreos/util.sh`, `kubernetes` is the value of `$POOL`. `$POOL_PATH` is the target dir for creating the pool named `$POOL` with `virsh`. Thus `${POOL_PATH}/kubernetes`should end up at `/opt/kubernetes` in the VM at the end of the day. 

I could have the wrong read on how this works, but this tracks with several other things in `cluster/libvirt-coreos/util.sh` - for example, `cluster/libvirt-coreos/util.sh` places the certs in `${POOL_PATH}/kubernetes/certs` and they are referenced from `cluster/libvirt-coreos/user_data_minion.yml` as `/opt/kubernetes/certs`.